src/components/YouboraAnalytics/InfoManager.brs

```diff
@@ -398,47 +412,48 @@ function InfoManager_getRequestParams(requestName = "" as string, params = inval
     if outParams.DoesExist("live") = false then outParams["live"] = m.getIsLive()
     if outParams.DoesExist("rendition") = false then outParams["rendition"] = m.getRendition()
     if outParams.DoesExist("title") = false then outParams["title"] = m.getTitle()
-    if outParams.DoesExist("properties") = false then outParams["properties"] = m.options["content.metadata"]
-    if outParams.DoesExist("cdn") = false then outParams["cdn"] = m.options["content.cdn"]
-    if outParams.DoesExist("program") = false then outParams["program"] = m.options["content.program"]
-    if outParams.DoesExist("saga") = false then outParams["saga"] = m.options["content.saga"]
-    if outParams.DoesExist("tvshow") = false then outParams["tvshow"] = m.options["content.tvShow"]
-    if outParams.DoesExist("season") = false then outParams["season"] = m.options["content.season"]
-    if outParams.DoesExist("titleEpisode") = false then outParams["titleEpisode"] = m.options["content.episodeTitle"]
-    if outParams.DoesExist("channel") = false then outParams["channel"] = m.options["content.Channel"]
-    if outParams.DoesExist("contentId") = false then outParams["contentId"] = m.options["content.id"]
-    if outParams.DoesExist("imdbID") = false then outParams["imdbID"] = m.options["content.imdbId"]
-    if outParams.DoesExist("gracenoteID") = false then outParams["gracenoteID"] = m.options["content.gracenoteId"]
-    if outParams.DoesExist("contentType") = false then outParams["contentType"] = m.options["content.type"]
-    if outParams.DoesExist("genre") = false then outParams["genre"] = m.options["content.genre"]
-    if outParams.DoesExist("contentLanguage") = false then outParams["contentLanguage"] = m.options["content.language"]
+    if outParams.DoesExist("properties") = false then outParams["properties"] = m.options.content["metadata"]
```

Comment:
please define a dictionary with keys and values  like 
{
contentId: id
contentLanguage: language
titleEpisode: episodeTile 
....
}
and then iterate these values in one for 
for each attribute in contentAttributes.items()
      outParams[attribute.key] = outParams[attribute.key] = invalid ?  m.content[attribute.val] : outParams[attribute.key]
end for

Reply:
Done,
Wanna ask a question. Do we need to change the logic to this one in the entire file or only in this case? In case the Youbora library is updated, we will need to load a new one and make these changes again

Reply:
@rdaci23